server/src/api/customer/customer.controller.ts

```diff
@@ -87,13 +96,32 @@ export const patchCustomer: RouteHandlerFnc = async (ctx) => {
         .$relatedQuery('representatives', trx)
         .whereIn('id', removed)
         .unrelate();
+
+      for (let representative of removed) {
+        await emitRoadmapEvent(ctx.io, {
+          roadmapId: Number(ctx.params.roadmapId),
+          onlyEmitToUserId: Number(representative),
+          event: ClientEvents.USERINFO_UPDATED,
+          eventParams: [],
+        });
+      }
```

Comment:
```suggestion
      await emitRoadmapEvent(ctx.io, {
        roadmapId: Number(ctx.params.roadmapId),
        onlyEmitToUserIds: removed,
        event: ClientEvents.USERINFO_UPDATED,
        eventParams: [],
      });
```

Having the "only id" be an array of ids instead would make this a lot nicer.
Also each iteration of the loop needs to do pretty much the same things,
so there will be a lot of repeated work that.

This also reduces the excessive `await`ing

Sending to a single user still very reasonable, just needs wrapping the one id to an array.